compiler/src/iree/compiler/Codegen/Common/TileDispatchUsingForall.cpp

```diff
@@ -190,6 +194,25 @@ pruneDroppedLoops(ArrayRef<Attribute> inputs,
   return prunedAttrs;
 }
 
+// Checks whether we have static dimension for all the loop bounds and steps.
+// This is a requirement if the reordering strategy is set to `transpose`.
+static bool checkStaticLoopBounds(scf::ForallOp forallOp) {
```

Comment:
style nit: use `///`

I think we can rename it to `areAllStaticLoopBounds`. It is clearer to me.